libs/clients/rsk/personal-tax-return/src/lib/transformers/transformers.ts

```diff
@@ -0,0 +1,86 @@
+import { PdfResponse, DirectTaxPaymentResponse } from '../responses'
+import { PdfDto, DirectTaxPaymentDto } from '../dto'
+
+export function pdfResponseToDto(response: PdfResponse): PdfDto {
+  const result =
+    response['s:Envelope']['s:Body'].SaekjaPDFAfritFramtalsEinstaklingsResponse
+      .SaekjaPDFAfritFramtalsEinstaklingsResult
+  return {
+    success: result['b:Tokst'],
+    errorText: result['b:Villubod'],
+    content: result['b:PDFAfritFramtals'],
+  }
+}
+
+export function directTaxPaymentResponseToDto(
```

Comment:
Wow this is a veisla 🥳

Reply:
Yup you don't want to know how long this took ⏱️  and I skipped some data 🙈